channeldb/db.go

```diff
@@ -245,6 +245,11 @@ var (
 	// channelOpeningState for each channel that is currently in the process
 	// of being opened.
 	channelOpeningStateBucket = []byte("channelOpeningState")
+
+	// channelFeeBucket is the database bucket used to store the
+	// channel fees for each permanent channel that is currently in the process
+	// of being opened.
+	channelFeeBucket = []byte("channelFees")
```

Comment:
Q: will these bucket be reused for other custom forward policies like minhtlc? It can be added in another PR (if needed) but I wonder if it is better to call the bucket something else like "channelInitialForwardPolicy"

Reply:
Yes, imo ideally all fields from `htlc.ForwardingPolicy` should be able to be specified from `lncli` and be announced and saved like the channel fees here. This enhancement would probably go well in another PR with the test refactoring above in `funding/manager_test.go`. I'll rename the bucket to `initialChannelForwardingPolicy`.